testing/tests/tests.py

```diff
@@ -988,6 +988,12 @@ def test_report_only_set(self):
             self.assertNotIn('Content-Security-Policy', response)
             self.assertIn('Content-Security-Policy-Report-Only', response)
 
+    def test_both_enforce_and_report_only(self):
+        with self.settings(CSP_MODE='enforce-and-report-only'):
+            response = self.client.get('/accounts/login/')
```

Comment:
I think this line should grab `LOGIN_URL` from django settings.

We all miss you @MartinPetkov

Reply:
Done.

I also changed `tox.ini` so now all the tests pass, including pep8.

Reply:
Miss you too. Say hi to everyone!